src/api.js

```diff
@@ -573,94 +554,21 @@ var API = function(options) {
-   * the world after breaking the API is easy.
-   */
   experimental:     'experimental',
-  /**
-   * API is stable and we will not delete resources or break the API suddenly.
-   * As a guideline we will always facilitate gradual migration if we change
-   * a stable API.
-   *
-   * **Intended Usage:**
-   *  - API end-points used in critical production.
-   *  - APIs so widely used that refactoring would be hard.
-   */
   stable:           'stable'
 };
 
 // List of valid stability-levels
 var STABILITY_LEVELS = _.values(stability);
 
 /**
- * Declare an API end-point entry, where options is on the following form:
- *
- * {
- *   method:   'post|head|put|get|delete',
- *   route:    '/object/:id/action/:param',      // URL pattern with parameters
- *   params: {                                   // Patterns for URL params
- *     param: /.../,                             // Reg-exp pattern
- *     id(val) { return "..." }                  // Function, returns message
- *                                               // if value is invalid
- *     // The `params` option from new API(), will be used as fall-back
- *   },
- *   query: {                                    // Query-string parameters
- *     offset: /.../,                            // Reg-exp pattern
- *     limit(n) { return "..." }                 // Function, returns message
- *                                               // if value is invalid
- *     // Query-string options are always optional (at-least in this iteration)
- *   },
- *   name:     'identifierForLibraries',         // identifier for client libraries
- *   stability: base.API.stability.experimental, // API stability level
- *   scopes:   ['admin', 'superuser'],           // Scopes for the request
- *   scopes:   [['admin'], ['per1', 'per2']],    // Scopes in disjunctive form
- *                                               // admin OR (per1 AND per2)
- *   input:    'input-schema.json',              // optional, null if no input
- *   output:   'output-schema.json' || 'blob',   // optional, null if no output
- *   skipInputValidation:    true,               // defaults to false
- *   skipOutputValidation:   true,               // defaults to false
- *   title:     "My API Method",
- *   description: [
- *     "Description of method in markdown, enjoy"
- *   ].join('\n'),
- *   cleanPayload: payload => payload,           // function to 'clean' the payload for
- *                                               // error messages (e.g., remove secrets)
- * }
- *
- * The handler parameter is a normal connect/express request handler, it should
- * return JSON replies with `request.reply(json)` and errors with
- * `request.json(code, json)`, as `request.reply` may be validated against the
- * declared output schema.
- *
- * **Note** the handler may return a promise, if this promise fails we will
- * log the error and return an error message. If the promise is successful,
- * nothing happens.
```

Comment:
I have mixed feelings about removing these docs from the source... It makes it really hard to write code.
I use this 100% when editing the code... Also If removing it I would like this example and description like format to maintained in the readme.

Or we should make the readme docs a table with: `name`, `type`, `example` and `description`... But I don't see how readme can nicely document things like `query`... This doc format is extremely concise and quick to read, can we please maintain it?